examples/docs_snippets/docs_snippets/guides/tutorials/dagster_tutorial/commands/dg-scaffold-custom-component.txt

```diff
@@ -1,3 +1,3 @@
-dg scaffold defs dagster_tutorial.components.tutorial.Tutorial tutorial
+dg scaffold defs my_project.components.tutorial.Tutorial tutorial
```

Comment:
Updating to the placeholder value, this is the only part that worries me since it's a copy and paste section and if the user is following the guide, this will lead to an error (instead of `dg scaffold defs dagster_tutorial.components.tutorial.Tutorial tutorial`)

@neverett